test/performance/performance.go

```diff
@@ -38,6 +39,11 @@ const (
 	istioNS      = "istio-system"
 	monitoringNS = "knative-monitoring"
 	gateway      = "istio-ingressgateway"
+	// Property name used by testgrid.
+	perfLatency = "perf_latency"
+	concurrency = 5
+	duration    = 1 * time.Minute
+	numThreads  = 1
```

Comment:
Is it really helpful to externalize `concurrency` this way?

For the existing tests in particular, `5` makes sense for the observed_concurrency test, because it's containerConcurrency is set to 1. For the latency test this will barely measure anything as it won't trigger any scale whatsoever.

That's not to say that it doesn't have any value at the concurrency of 5, I feel like these are values to be set per test though.

Reply:
I wanted to add some default value for these and the tests can add their own if needed. My plan was to update these after talking to the scaling WG and seeing what values make sense.

What do you think should be values for latency tests :) ?